src/Plugin/Field/FieldWidget/CaptchaWidget.php

```diff
@@ -36,7 +36,7 @@ public static function processCaptcha(array &$element, FormStateInterface $form_
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     // FIXME: Load this from a storage.
     $captcha = [
-      'question' => t('What is a common pet that eats mice?', ['context' => 'Captcha question']),
+      'question' => t('Captcha: What is a common pet that eats mice?', ['context' => 'Captcha question']),
```

Comment:
Should this be CAPTCHA, since it's an acronym, instead? I don't know if it's the case in practice but occurred to me that some text to speech engines might read it letter by letter if it's capitalized and not do that otherwise so it could be best to have it in the same form that is most commonly used for consistent reading.

Reply:
Yes it should. Here's the fixed commit.